series/olomouc-pyvo/series.yaml

```diff
@@ -0,0 +1,20 @@
+name: Olomoucké Pyvo
+city: olomouc
+description:
+  cs: |
+    Olomoucké Pyvo bývá každý druhý čtvrtek v měsíci.
+  en: |
+    Olomouc Pyvo is traditionally held on the second Thursday of each month.
+organizer-info:
+  - name: Martin Čurlej
+    phone: 739246296
+    mail: martin.curlej@gmail.com
+  - name: Filip Valder
+    phone: 608123966
+    mail: filip.valder@gmail.com
+recurrence:
+  scheme: monthly
+  rrule: RRULE:FREQ=MONTHLY;INTERVAL=1;BYDAY=2TH;BYHOUR=18;BYMINUTE=0;BYSECOND=0
+  description:
+    cs: Olomoucké Pyvo bývá každý druhý čtvrtek v měsíci.
```

Comment:
Rovnou slibujeme opakování?

Reply:
Tak trochu s tím počítám, i kdyby to mělo být v neoficiální podobě. Ale finálě samozřejmě záleží na Martinovi a Filipovi.

Reply:
"Kdo se bojí, nesmí do lesa" 😉  Za mě 👍